test/e2e/fixtures/lint/file-with-lint-warnings.ts

```diff
@@ -0,0 +1,4 @@
+const foo = () => {
+  const bar = 'baz';
+  const foobar = '';
+};
```

Comment:
```suggestion
// this file should have 3 unused var lint warnings
const unusedVar1 = () => {
  const unusedVar2 = 'baz';
  const unusedVar3 = '';
};
```

The lint fixtures don't seem to have comments, but some other fixtures do and it would be useful to have so it's easier to interpret at-a-glance. Changed naming to be similarly easier to interpret rather than random